parsers/objc.c

```diff
@@ -23,17 +23,6 @@
 #include "selectors.h"
 #include "vstring.h"
 
-/* To get rid of unused parameter warning in
- * -Wextra */
-#ifdef UNUSED
-#elif defined(__GNUC__)
-# define UNUSED(x) UNUSED_ ## x __attribute__((unused))
-#elif defined(__LCLINT__)
-# define UNUSED(x) /*@unused@*/ x
-#else
-# define UNUSED(x) x
-#endif
```

Comment:
Ah, I haven't noticed that, will fix.

But that means we had it wrong in Geany because there the global macro is called UNUSED too so all these different definitions get skipped by the first ifdef.